README.md

```diff
@@ -1,2 +1,84 @@
-# jesi_dart_analysis
-This package provides lint rules for Dart and Flutter which are used at JESI
+# JESI Dart Analysis
+
+This package provides lint rules for Dart and Flutter which are used at JESI.
+
+For more information on each option, see the complete [list of options](https://dart.dev/tools/linter-rules).
+
+For more information about the core libraries, see the related package:
+
+- [Dart Lints](https://pub.dev/packages/lints)
+- [Flutter Lints](https://pub.dev/packages/flutter_lints)
+
+## Usage
+
+To use the linting options, add a dependencies into your `pubspec.yaml`:
+
+```yaml
+dev_dependencies:
+  lints: '>=1.0.1 <2.0.0' # Add for Dart
+  # flutter_lints: '>=1.0.0 <2.0.0' # Add for Flutter
```

Comment:
lint packages are excluded from here. I don't see the point of adding when they should be dev dependencies.

Also, we can't separate flutter from dart unless we create sub packages.

Reply:
If you try to use this package without the relevant lint dev dependency, warnings about the missing dependency is given.